client/src/containers/Corruption/CorruptionView.tsx

```diff
@@ -57,18 +64,20 @@ export const CorruptionView: FC<CorruptionViewProps> = ({ corruption }) => {
       gap={4}
       mb={4}
     >
-      <GridItem w="100%" onClick={() => setIsScoreActive(false)}>
-        <Box mb={4} cursor={'pointer'}>
+      <GridItem w="100%">
+        <Box mb={4} cursor={'pointer'}  onClick={() => {setIsScoreActive(false); setSelectedId(CORRUPTION_INDEX_IDS["CORRUPTION_RATING"]); }}>
           <Card // TODO calculate the total number of countires instead of 180
             value={<Action>{`${ukraineThisYear.rank} / 180`}</Action>}
             title={t('corruption.rank')}
+            isActive={selectedId === CORRUPTION_INDEX_IDS["CORRUPTION_RATING"]}
           />
         </Box>
 
-        <Box cursor={'pointer'}>
+        <Box cursor={'pointer'}  onClick={() => {setIsScoreActive(false); setSelectedId(CORRUPTION_INDEX_IDS["CORRUPTION_POSITION"]); }}>
```

Comment:
There were no this handler before. What is the reason of addin it here?

Reply:
where should I add it instead?

Reply:
@MaksymRudnyi